types/koa-session/index.d.ts

```diff
@@ -108,7 +112,7 @@ declare namespace session {
         hash(sess: any): string;
     }
 
-    interface opts {
+    interface opts extends Omit<Cookies.SetOption, 'maxAge'> {
```

Comment:
You can consider changing to `Omit<T, K extends keyof T>`. That way the string "maxAge" is now tied to the property maxAge on Cookies.SetOption (so if they change the name or remove it then this library will break too and you can make the appropriate changes here).